### PR TITLE
fix alloy metrics app catalog

### PR DESCRIPTION
**Pull request by QuentinBisson**
Fix alloy metrics app catalog

This PR:

* adds/changes/removes etc

### Checklist

- [ ] Update changelog in CHANGELOG.md.
- [ ] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

**Review by hervenicol — COMMENTED** [review]
Do we really need to use the `test` catalog, and not the proper `giantswarm` catalog?

File: helm/observability-bundle/values.yaml [hunk 1]
```diff
@@ -68,7 +68,7 @@ apps:
   alloyMetrics:
     appName: alloy-metrics
     chartName: alloy
-    catalog: giantswarm-test
+    catalog: giantswarmtes
```
Comment (QuentinBisson) [inline helm/observability-bundle/values.yaml:71]:
```suggestion
    catalog: giantswarm
```

**Comment by QuentinBisson** [comment]
Alloy was released, I think it was an oversight as Théo was testing a dev version for metrics

**Review by hervenicol — APPROVED** [review]
Actually it's only addressing my semi-unrelated comment to the initial PR?
LGTM then :man_shrugging: :laughing:

**Pull request merged**